core/src/banking_stage.rs

```diff
@@ -611,7 +615,12 @@ impl BankingStage {
         let result = bank.check_transactions(
             transactions,
             &filter,
-            (MAX_RECENT_BLOCKHASHES - MAX_TRANSACTION_FORWARDING_DELAY) / 2,
+            (MAX_RECENT_BLOCKHASHES / 2)
+                .saturating_sub(MAX_TRANSACTION_FORWARDING_DELAY)
+                .saturating_sub(
+                    (FORWARD_TRANSACTIONS_TO_LEADER_AT_SLOT_OFFSET * DEFAULT_TICKS_PER_SLOT
```

Comment:
Can we grab the actual values from the fullnodeconfig rather than using default values? Some of the tests don't use default values.

Reply:
Can we also add a comment explaining the reasoning behind this computation?

Reply:
👍

Reply:
There are also a bunch of other default values used in this file, can we clean them up while we're at it (the list of ones we use are the ones we import in the header)